local_planner/src/nodes/common.h

```diff
@@ -10,58 +10,53 @@
 
 namespace avoidance {
 
-float distance2DPolar(int e1, int z1, int e2, int z2);
+struct PolarPoint {
+  float e;
+  float z;
+  float r;
+};
+
+float distance2DPolar(const PolarPoint& p1, const PolarPoint& p2);
```

Comment:
Since this function is quite obvious, it doesn't need lots of comments, but a `@brief` with one line would be nice